cypress/support/commands.ts

```diff
@@ -107,7 +107,6 @@ Cypress.Commands.add(
 )
 
 // login via the purple OSS screen by typing in username/password
-// this is only used if you're using monitor-ci + DEV_MODE_CLOUD=0
```

Comment:
TODO: This function itself may also be legacy. The login() methods may be cleaned up further, once we get the oss-local e2es running.